docs/src/main/tut/docs/combinators.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+title: Combinators
+---
+
+## {{page.title}}
+
+The combinators defined in `ConfigReader` provide an easy way to create new `ConfigReader` instances by transforming
```

Comment:
I suggest linking from `ConfigReader` to the [source](https://github.com/pureconfig/pureconfig/blob/master/core/src/main/scala/pureconfig/ConfigReader.scala). Normally I'd want to link to the scaladoc, but we have not published it, so I think the source is our best option.